File: shoring/flywheel/src/test/java/com/panopset/tests/flywheel/DepthTest.java
package com.panopset.tests.flywheel;

import java.io.IOException;
import org.junit.jupiter.api.Test;

public final class DepthTest {

  @Test
  void testDepth() throws IOException {
    new SimpleTest().comparisonTest("depthTest.txt", "depthTest01.txt",
        "depthTest01Expected.txt");
    new SimpleTest().comparisonTest("depthTest.txt", "depthTest02.txt",
        "depthTest02Expected.txt");
  }

  @Test
  void testDeeper() throws IOException {
    new SimpleTest().comparisonTest("deepTest.txt", "deepTest.txt",
        "deepTestExpected.txt");
  }
}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/flywheel/DepthTest.kt
package com.panopset.tests.flywheel

import org.junit.jupiter.api.Test

class DepthTest {
    @Test
    fun testDepth() {
        SimpleTest().comparisonTest(
            "depthTest.txt", "depthTest01.txt",
            "depthTest01Expected.txt"
        )
        SimpleTest().comparisonTest(
            "depthTest.txt", "depthTest02.txt",
            "depthTest02Expected.txt"
        )
    }

    @Test
    fun testDeeper() {
        SimpleTest().comparisonTest(
            "deepTest.txt", "deepTest.txt",
            "deepTestExpected.txt"
        )
    }
}
